fastlane/actions/podspec_constant_versioning.rb

```diff
@@ -0,0 +1,78 @@
+module Fastlane
+  module Actions
+    class PodspecConstantVersioningAction < Action
+      def self.run(params)
+        podspec = params[:podspec]
+        constant = params[:constant]
+        value = params[:value]
+        
+        podspec_contents = File.read(podspec)
+
+        # TODO: improve RegEx
+        if !podspec_contents.match(/(?<=#{constant})(\s*=\s*["'].*)/)
+          UI.error("#{constant} not present or doesn't have an explicit value in #{podspec}")
+          return
+        end
+
+        new_value = " = '#{value}'"
+        
+        podspec_new_contents = podspec_contents.gsub(/(?<=#{constant})(\s*=\s*["'].*)/, new_value)
+
+        File.open(podspec, "w") { |file| file.puts podspec_new_contents }
+        UI.success("successfully modified #{constant} to value #{value} in #{podspec}")
+            
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "This action will modify the value of a constant in your podspec."
```

Comment:
This is more general: it looks to me like it will change the value of any `key=value` pattern from any file?